### DISCUSSION TITLE
Curso Excelente

### DISCUSSION THREAD

**Discussion by giannycabral** [skills/.github#732] [Introduction to GitHub]
Curso muito bom, super bem explicado e mão na massa. Adorei <3

**Comment by Georgiaava** [comment]
The feminine noun among the options provided is 'Clase', which means 'class' in Spanish. The other options 'Amigo', 'Curso', and 'Continente' are masculine nouns. This distinction is important in  [monkey type ](https://monkey-type.net/) Spanish as it affects article usage and adjective agreement.The feminine noun among the options provided is 'Clase', which means 'class' in Spanish. The other options 'Amigo', 'Curso', and 'Continente' are masculine nouns. This distinction is important in Spanish as it affects article usage and adjective agreement.
